openpype/tools/mayalookassigner/app.py

```diff
@@ -147,11 +155,14 @@ def remove_connection(self):
 
     def showEvent(self, event):
         self.setup_connections()
-        super(App, self).showEvent(event)
+        super(MayaLookAssignerWindow, self).showEvent(event)
+        if self._first_show:
+            self._first_show = False
+            self.setStyleSheet(style.load_stylesheet())
```

Comment:
Why do we need this additional set stylesheet call? Seems something is off? It's already being set [here](https://github.com/pypeclub/OpenPype/pull/2889/files#diff-24a0bf91d45dc87a044e19664ed3f6271bf0544dd6b300e650095a8d20bbe9e0R281) no?

Reply:
The `show` function is not used in Maya. Removed the setStyleSheet there.